CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+**Unreleased**
+
+* :meth:`Emoji.update` support updating an emoji's permissions
```

Comment:
As below, for proper Sphinx style, please make this read

:meth:`~.Emoji.update`

In addition, please ensure correct grammar by changing "support" to "supports" and adding a period at the end of the line.